pkg/operator/certrotation/cabundle.go

```diff
@@ -59,6 +59,7 @@ func (c CABundleConfigMap) EnsureConfigMapCABundle(ctx context.Context, signingC
 			c.AdditionalAnnotations,
 		)}
 		modified = true
+		exists = false
 	}
 
 	needsOwnerUpdate := false
```

Comment:
and we could modify line `70` to: 

`updateRequired = needsOwnerUpdate || needsMetadataUpdate`